scripts/plotting/tob_wgs_snp_chip_plot_pca/plot_pca_tob_wgs_snp_chip_datasets.py

```diff
@@ -0,0 +1,86 @@
+"""Create PCA plots for the combined
+TOB-WGS/SNP-chip data"""
+
+import subprocess
+from bokeh.io.export import get_screenshot_as_png
+from bokeh.plotting import output_file, save
+import pandas as pd
+import hail as hl
+import click
+from analysis_runner import output_path
+
+SCORES = 'gs://cpg-tob-wgs-main/tob_wgs_snp_chip_variant_pca/v6/scores.ht/'
+EIGENVALUES = 'gs://cpg-tob-wgs-main/tob_wgs_snp_chip_variant_pca/v6/eigenvalues.ht'
```

Comment:
Do you want to use `bucket_path` here?

```suggestion
SCORES = 'gs://cpg-tob-wgs-main/tob_wgs_snp_chip_variant_pca/v6/scores.ht/'
EIGENVALUES = 'gs://cpg-tob-wgs-main/tob_wgs_snp_chip_variant_pca/v6/eigenvalues.ht'
```

Reply:
I changed this to use the `bucket_path()` convenience function